docs/selectively-triggering-aktualizr.adoc

```diff
@@ -58,3 +58,20 @@ Installation reports are sent when aktualizr polls the server for updates, so th
 
     aktualizr --running-mode=check
 
+=== Check for campaigns target the device
+
+In the OTA Connect web app, you can create smart groups that filter for devices based on specific criteria. When OTA Connect detects a new device that matches this criteria, it automatically publishes the campaign to the device. This running mode checks to see if the device matches the filter criteria used in an active campaign.
+
+    aktualizr --running-mode=campaign_check
+
+If the device matches one or more campaigns, aktualizr prints details of each applicable campaign including the campaign ID. You can then use the campaign ID for the `campaign_accept` command.
+
+=== Accept campaigns that target the device
+
+This running mode works together with the `campaign_check` mode. After you have received a list of applicable campaigns, you can accept specific campaigns so that they update the device.
+
+    aktualizr --running-mode=campaign_accept <CAMPAIGN_ID>
```

Comment:
The format is actually:

    aktualizr --running-mode=campaign_accept --campaign-id=<CAMPAIGN_ID>

Reply:
Ok fixed